package.json

```diff
@@ -4,8 +4,10 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "test": "jest",
+    "test": "npm run test:component && npm run test:contract",
     "test:watch": "jest --watch",
+    "test:component": "jest --testPathPattern=./test/component/",
+    "test:contract": "jest --testPathPattern=./test/contract/ --runInBand",
```

Comment:
use a configuration file, for example `jest.contract.js` instead of pass parameter on the script